std_msgs/Header   header

uint16 seq

int8 roll_torque_percent
int8 pitch_torque_percent
int8 yaw_torque_percent

int16 roll_velocity
int16 pitch_velocity
int16 yaw_velocity

float32 relative_roll
float32 relative_pitch
float32 relative_yaw

float32 target_roll
float32 target_pitch
float32 target_yaw

float32 actual_roll
float32 actual_pitch
float32 actual_yaw